extensions/gmsa-coredns/v1/update-coredns.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Get the IP of the DC as soon as it is available
+while [$DCIP = $null]
+do
+    DCIP=$(kubectl get node --selector="agentpool=windowsgmsa" -o jsonpath="{.items[*].status.addresses[?(@.type=='InternalIP')].address}") > /dev/null 2>&1
```

Comment:
This extension should run on the master node, correct? The Windows agents start some 10-15 minutes later than the master ( considering we are also prepulling test images ). I wonder if this extension might timeout waiting for the agentpool node to be registered?

Reply:
If you don't apply the Windows Udpate extension to the gmsa pool, it comes up rather quickly.  This part of the extension finds the IP of the Domain Controller.